wolfcrypt/src/ecc.c

```diff
@@ -6324,6 +6348,22 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
    }
    /* valid signature if we get to this point */
    *res = 1;
+#elif defined(WOLFSSL_SILABS_SE_ACCEL)
+   /* Extract R and S */
+
+   err = mp_to_unsigned_bin(r, &sigRS[0]);
+   if (err != MP_OKAY) {
+       return err;
+   }
+   err = mp_to_unsigned_bin(s, &sigRS[keySz]);
+   if (err != MP_OKAY) {
+       return err;
+   }
+
+   silabs_ecc_verify_hash(&sigRS[0], keySz*2,
```

Comment:
Is there a return code we can add here? I assume "res" is being used to indicate success, but we should also have an err return.